data/strings/strings.txt

```diff
@@ -16383,44 +16383,48 @@
-    de = Ihre Änderungsvorschläge wurden gesendet
-    el = Οι προτεινόμενες αλλαγές σας έχουν σταλεί
-    es = Se han enviado sus cambios sugeridos
-    et = Sinu soovitatud muudatused on saadetud
-    eu = Iradokitako aldaketak bidali dira
-    fa = تغییر پیشنهادی شما ارسال شد
-    fi = Ehdottamasi muutokset on lähetetty
-    fr = Vos modifications suggérées ont été envoyées
-    hu = Javasolt változtatásainak elküldése sikerült
-    id = Saran perubahan Anda telah dikirim
-    it = Le modifiche suggerite sono state inviate
-    ja = あなたの提案の変更が送信されました
-    ko = 귀하의 제안 변경사항이 전송되었습니다
-    lt = Jūsų pasiūlyti pakeitimas išsiųsti
-    mr = तुम्ही सुचवलेले बदल पाठवले आहेत
-    nb = De foreslåtte endringene er sendt
-    nl = Uw voorgestelde wijzigingen zijn verzonden
-    pl = Twoje sugestie zostały wysłane
-    pt = As suas alterações sugeridas foram enviadas
-    pt-BR = Suas sugestões de mudança foram enviadas
-    ro = Modificările sugerate au fost trimise
-    ru = Предложенные вами изменения отправлены
-    sk = Vykonali sa vami odporúčané zmeny
-    sv = Dina föreslagna ändringar har skickats
-    th = คำแนะนำการเปลี่ยนแปลงของคุณถูกส่งไปแล้ว
-    tr = Önerdiğiniz değişiklikler gönderildi
-    uk = Зміни, що ви запропонували, відправлено
-    vi = Những thay đổi đề nghị của bạn đã được gửi
-    zh-Hans = 您的建议已发送
-    zh-Hant = 您的建議已傳送
+    en = Your note will be sent to OpenStreetMap
+    af = Jou nota sal na OpenStreetMap gestuur word
+    ar = سيتم إرسال ملاحظتك إلى OpenStreetMap
+    be = Ваша нататка будзе адпраўлена ў OpenStreetMap
+    bg = Вашата бележка ще бъде изпратена до OpenStreetMap
+    ca = La teva nota s'enviarà a OpenStreetMap
+    cs = Vaše poznámka bude odeslána do OpenStreetMap
+    da = Din note vil blive sendt til OpenStreetMap
+    de = Ihre Notiz wird an OpenStreetMap gesendet
+    el = Η σημείωσή σας θα σταλεί στο OpenStreetMap
+    es = Tu nota se enviará a OpenStreetMap
+    es-MX = Tu nota será enviada a OpenStreetMap
+    et = Teie märkus saadetakse OpenStreetMapile
+    eu = Zure oharra OpenStreetMap-era bidaliko da
+    fa = یادداشت شما به OpenStreetMap ارسال خواهد شد
+    fi = Huomautus lähetetään OpenStreetMapille
+    fr = Ta note sera envoyée à OpenStreetMap
+    he = ההערה שלך תישלח אל OpenStreetMap
+    hi = आपका नोट OpenStreetMap पर भेजा जाएगा
+    hu = Az Ön megjegyzése elküldésre kerül az OpenStreetMap-nek
+    id = Catatan Anda akan dikirim ke OpenStreetMap
+    it = La tua nota sarà inviata a OpenStreetMap
+    ja = あなたのメモはOpenStreetMapに送られる
+    ko = 메모가 오픈스트리트맵으로 전송됩니다
+    lt = Jūsų pastaba bus išsiųsta į OpenStreetMap
+    mr = तुमची टीप OpenStreetMap वर पाठवली जाईल
+    nb = Notatet ditt vil bli sendt til OpenStreetMap
+    nl = Je opmerking wordt verzonden naar OpenStreetMap
+    pl = Twoja notatka zostanie wysłana do OpenStreetMap
+    pt = A tua nota será enviada para o OpenStreetMap
+    pt-BR = Sua nota será enviada ao OpenStreetMap
+    ro = Nota dvs. va fi trimisă către OpenStreetMap
```

Comment:
Romanian translation sounds good, I'm just curious about the vocabulary (across all languages): is this a note, a comment, a message?

Reply:
@vgrigoriu can you please check how is it called at osm.org? It's a note/comment to OSM editors what should be checked/fixed in the maps data.

Reply:
Yep, it's "note" on osm.org.

Reply:
"Hi" is good